### PR TITLE
chore: update testnet to lisbon-3

### PR DESCRIPTION
**Pull request by milapsheth**

### CODE REVIEW

File: resources/testnet-releases.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 Variable  | Value
 ------------- | -------------
-`axelar-core` version | `v0.13.7`
+`axelar-core` version | `v0.13.6`
 `tofnd` version | `v0.8.2`
 Ethereum Axelar Gateway contract address | `0x81b450CA8F0e842EcC361FD64270aeb5A374A0d8`
```
Comment (milapsheth) [inline resources/testnet-releases.md:8]: Need to update contract addresses

Comment (ggutoski) [reply resources/testnet-releases.md:8]: should probably merge now and do contract addresses in a separate PR so people can join testnet immediately.

**Review by ggutoski — APPROVED** [review]
+1 for "Proceeding with existing data..." msg

File: resources/testnet-releases.md (hunk 1 above)
Comment (milapsheth) [reply resources/testnet-releases.md:8]: Adding the ethereum/terra values since we need to set it up

**Comment by milapsheth** [comment]
Need to update seed to `a005b8923888007eb5cf9ed8c8120ed956bc31f7`

**Pull request merged**